rpcs3/Emu/Cell/lv2/sys_sync.h

```diff
@@ -110,18 +110,18 @@ struct lv2_obj
 
 	// Find and remove the object from the container (deque or vector)
 	template <typename T, typename E>
-	static T* unqueue(std::deque<T*>& queue, E* object)
+	static typename T::value_type unqueue(T& queue, E object)
```

Comment:
It's better to match as `std::deque<T>& queue`, and the comment about vector above is outdated. Vector is too slow for this purpose anyway.

Reply:
Fixed.